<launch>
  <node name="ar_docking" pkg="ar_docking" type="ar_docking" respawn="false"
    output="screen">
    <param name="marker_pattern" type="string"
      value="$(find ar_pose)/data/4x4/4x4_77.patt"/>
    <param name="marker_width" type="double" value="205.0"/>
    <param name="marker_center_x" type="double" value="0.0"/>
    <param name="marker_center_y" type="double" value="0.0"/>
    <param name="threshold" type="int" value="100"/>
    <param name="use_history" type="bool" value="true"/>
    <param name="filter_lambda" type="double" value="0.2"/>
    <param name="filter_kt" type="double" value="-2.0"/>
    <param name="filter_kx" type="double" value="-0.05"/>
    <param name="cam_info_file" type="string" value="file:///home/rob/.ros/camera_info/camera.yaml"/>  
    <param name="cmd_vel_topic" type="string" value="/kobra/locomotion_cmd_vel2"/>
    <param name="camera_topic" type="string" value="/docking_cam/image_raw"/>
    <param name="power_info_topic" type="string" value="/npb/power_info"/>
    <param name="start_stop_service_name" type="string" value="docking_service"/>
  </node>

  <node name="docking_cam" pkg="usb_cam" type="usb_cam_node" respawn="false"
  </node>

</launch>
